src/main/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeRepository.java

```diff
@@ -75,6 +76,16 @@ public EnvelopeRepository(NamedParameterJdbcTemplate jdbcTemplate, EnvelopeMappe
         );
     }
 
+    public List<Envelope> find(Instant from, Instant to) {
```

Comment:
maybe all we need is one parameter - date? (without the time)

Reply:
One parameter - date - will be in a service method, this method will convert the date into the from - to range

Reply:
if you think we'll need range somewhere else...